CONTRIBUTING.md

```diff
@@ -1,5 +1,14 @@
 Hello you,
 
+## Install the scrapper without Docker
+
+We do recommend the use of [pipenv][1] to install the whole python environment
+
+- [Install pipenv][2]
+- `pipenv --python 2.7 install`
+
```

Comment:
And  `pipenv shell` to drop into a shell with all the required dependencies, or `pipenv run ./docsearch` to start docsearch